build/GenerateCppJsRulesDescription.ps1

```diff
@@ -0,0 +1,65 @@
+param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "target path to store the rules description")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $targetFolder,
+
+    [Parameter(Mandatory = $true, HelpMessage = "path to rule extractor (from sonarlint-core). For example 'c:\src\sonarlint-rule-extractor-2.18-SNAPSHOT-jar-with-dependencies.jar' ")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $ruleExtractor
+)
+
+function FindDaemonVersion($fileText)
```

Comment:
Let's try to match Powershell syntax: `Find-DaemonVersion`. See the list of approved powershell verbs: https://blogs.msdn.microsoft.com/powershell/2009/07/15/final-approved-verb-list-for-windows-powershell-2-0/.

Let's do the other method names too.